src/app/pages/loading-waiting-states/skeleton-loader/skeleton-loader.component.html

```diff
@@ -12,10 +12,10 @@
 <section class="mat-typography" [class.small]="isSmall">
     <div class="score-card-example-section">
         <h2 class="example-title">Scorecard</h2>
-        <div class="placeholder-section" *ngIf="showPlaceHolder">
+        <div class="placeholder-section" [hidden]="!showPlaceHolder">
             <app-placeholder-score-card></app-placeholder-score-card>
         </div>
-        <div class="loaded-section" *ngIf="!showPlaceHolder">
+        <div class="loaded-section" [hidden]="showPlaceHolder">
             <mat-card>
```

Comment:
Our scorecard should not be used inside a mat-card. Now we have two layers of shadows.